### DISCUSSION TITLE
How long are the Github Copilot wait times?

### DISCUSSION THREAD

**Discussion by itamar1238** [community/community#23504] [New to GitHub]
<p>Hey, I applied for github co-pilot about 3 months ago and Im still on the waiting list. Just wondering how much longer it will take.</p>
<p>Thanks!</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>As GitHub Copilot utilizes our state-of-the-art AI hardware, we are offering it to a limited number of testers for free, during the technical preview. We want to use the preview to learn how people use GitHub Copilot and what it takes to operate it at scale. If the technical preview is successful, our plan is to build a commercial version of GitHub Copilot in the future. As a result, the team is cautiously adding users to the technical preview as the product is refined. <em><strong>Users are not added on a first come first serve basis</strong></em> as we want to ensure a diverse group of users is testing the program. I understand how that can be frustrating, but this is in the hopes that one day this will be available to all users publicly so we really appreciate everyone’s patience.</p>
<p>Please know GitHub Support and the Community Forum are unable to expedite this process for you, but there is a <a href="https://github.community/t/copilot-waitlist/188378/">thread</a> for you to discuss the waitlist with fellow users.</p>

**Comment by Mast3rMast3r** [comment]
<p>im still waiting hope i can get it</p>

**Comment by kamrankiyani7777** [comment]
unable to see chat icon on VS code even i have subscribed for it currently on free trail

**Reply by TristanLib** [reply]
You should use vscode inside

**Reply by agilexmarketing** [reply]
huh?
